benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_uperf_logs/analyze_uperf_logs.py

```diff
@@ -1,36 +1,36 @@
-import csv
+
 import os
 import pandas as pd
-
-# display df
-from IPython.display import display
+from typeguard import typechecked
 
 # display bokeh
 from bokeh.plotting import figure, show
 from bokeh.io import output_notebook, output_file
-from bokeh.models import HoverTool, NumeralTickFormatter
+from bokeh.models import HoverTool, Label, ColumnDataSource, LabelSet, Circle, NumeralTickFormatter
 
-from typeguard import typechecked
 
 from benchmark_runner.jupyterlab.templates.logs_operations.logs_operations import LogsOperations
+from benchmark_runner.jupyterlab.templates.analyze_workloads.analyze_workloads_operations import AnalyzeWorkloadOperations
 
 
-class AnalyzeUperfLogs:
+class AnalyzeUperfLogs(AnalyzeWorkloadOperations):
```

Comment:
Remove **AnalyzeWorkloadOperations** this.
As you created an object no need to extend it here.

Reply:
Fixed, using inheritance